src/extensions/contact-loaders/ngpvan/index.js

```diff
@@ -108,7 +108,7 @@ export async function getClientChoiceData(organization, campaign, user) {
       DEFAULT_NGP_VAN_MAXIMUM_LIST_SIZE;
 
     const url = Van.makeUrl(
-      `v4/savedLists?$top=&maxPeopleCount=${maxPeopleCount}`,
```

Comment:
Let's match the MoveOn fix to this bug in their PR: https://github.com/MoveOnOrg/Spoke/pull/2024